CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+- Add ROS 2 Kilted to CI jobs ([#2719](https://github.com/stack-of-tasks/pinocchio/pull/2719))
+
```

Comment:
```suggestion
```

We usually don't add CI related development in the changelog.

Reply:
Sorry, the automated workflow asked for one

Reply:
Don't be sorry, it's not well documented ;)